skeletons/web-extension/boot.js

```diff
@@ -3,6 +3,7 @@
 // TODO: make this conditional, only do it when requested.
 // only inject boot script when on an HTML page
 if (document.contentType === 'text/html') {
+  window.EmberENV = { _DEBUG_RENDER_TREE: true };
```

Comment:
Wouldn't we want to check for existence of `window.EmberENV` here too instead of setting it to our own object?

Reply:
Not needed here. This really absolutely runs before anything else